### DISCUSSION TITLE
Is it possible to invalidate/clear cache for an exiting file on Amazon S3?

### DISCUSSION THREAD

**Discussion by evimeyer** [iterate-ch/cyberduck#14445] [Q&A] [answered]
Hello,

Is there a quick way in CyberDuck to invalidate/clear cache for an exiting file/folder?

Thanks!

**Comment by evimeyer** [comment, marked as answer]
Solution found:

1. Right click on the file/folder
2. Click "Info"
3. Go to CDN tab and configure

**Comment by dkocher** [comment]
Documentation for Amazon CloudFront can be found in the [documentation](https://docs.cyberduck.io/protocols/cdn/cloudfront/).
